tests/testthat/test-clean_data.R

```diff
@@ -5,14 +5,31 @@ DAT <- seq(as.Date("2018-01-01"), length.out = 10, by = "1 day")
 POS <- as.POSIXlt(DAT, tz = "UTC")
 POS2 <- POS + (86400 * sample(10, 10, replace = TRUE)) + sample(86400, 10)
 md  <- messy_data(10)
+md$`'ID` <- as.character(md$`'ID`)
 md$`!!Date of Admission` <- DAT
 md$`Time of admission`   <- POS
 md$`Time of   discharge` <- POS2
 
+
+# location data with mis-spellings, French, and English.
+messy_locations <- c("hopsital", "h\u00f4pital", "hospital", 
+                     "m\u00e9dical", "clinic", 
+                     "feild", "field", "hopsital", 
+                     "home", "m\u00e9dical")
+md$location <- factor(messy_locations)
+
+# add a wordlist
+wordlist <- data.frame(
+  from  = c("hopsital", "hopital",  "medical", "feild"),
+  to    = c("hospital", "hospital", "clinic",  "field"),
+  var_shortname = rep("location", 4),
```

Comment:
maybe just rename to `variable`? It's just `shortname` may suggest there's a `longname` too and get people confused.

Reply:
will do. I'll also show an example where there is a longname since that's what the MSF dictionaries use.